src/main/java/de/eposcat/master/model/Page.java

```diff
@@ -2,30 +2,32 @@
 
 import java.util.HashMap;
 import java.util.Map;
+import java.util.Objects;
 
 public class Page {
-    private int id;
+    private long id;
+    //TODO make typeName mandatory
```

Comment:
Not sure, currently the type is not used as a grouping for attributes. One option would be to remove it because it is not used. Another would be to rename it to pageName, which seems to be more in-line with the current implementation. This TODO: should be tackled in a separate PR.